site/content/extend/plugins/example-plugins.md

```diff
@@ -28,6 +42,16 @@ Topics demonstrated:
 * Implements tests using the [plugin/plugintest](https://godoc.org/github.com/mattermost/mattermost-server/plugin/plugintest) package.
 * Compiles and publishes releases for multiple platforms using Travis-CI.
 
+## Profanity Filter
+
+The [profanity filter plugin for Mattermost](https://github.com/mattermost/mattermost-plugin-profanity-filter) automatically detects restricted words in posts and censors them prior to writing to the database. For more use cases, [see this forum post](https://forum.mattermost.org/t/coming-soon-apiv4-mattermost-post-intercept/4982).
+
+Topics demonstrated:
+
+* // XX @crspeller perhaps you can help fill this in?
```

Comment:
@crspeller perhaps you can help fill this in?

Reply:
- Interception and modification of posts.
- Rejection of posts.